content/src/fieldConfig/navigation-defaults.json

```diff
@@ -0,0 +1,49 @@
+{
+  "navigationDefaults": {
+    "navBarUnnamedDbaBusinessText": "Unnamed DBA",
+    "landingPageDropDownTitle": "My Account",
+    "navBarUnnamedOwnedBusinessText": "Unnamed Business",
+    "needsAccountSnackbarTitle": "Link with myNewJersey to Save Your Progress",
+    "accountSuccessSnackbarTitleExistingAccount": "Your myNewJersey Account is Linked",
+    "navBarBusinessNJGovAriaLabel": "Business.nj.gov Home",
+    "needsAccountSnackbarBodyExistingAccount": "Now that you’ve filled out your business information, you can save your progress and come back later by [linking your myNewJersey account](/account-setup?source=guest_snackbar). ",
+    "navBarGuestRegistrationText": "Link with myNewJersey",
+    "needsAccountModalSubText": "Already have an account? [Log In](#)",
+    "profileLinkText": "Business Profile",
+    "myNJAccountText": "myNewJersey account",
+    "addBusinessButton": "Add a Business",
+    "navBarBusinessNJGovAltText": "Business.nj.gov Logo",
+    "navBarUnnamedForeignRemoteSellerWorkerText": "Unnamed Out-of-State Business",
+    "navBarGuestBusinessText": "Guest Business",
+    "navBarGuestAccountText": "Guest Account",
+    "getStartedText": "Get Started",
+    "needsAccountSnackbarBody": "Now that you've filled out your business information, you can save your progress [link your Business.NJ.gov account with myNewJersey](/account-setup?source=guest_snackbar). ",
+    "logInButton": "Log in",
+    "needsAccountModalTitle": "Link with myNewJersey to Save Your Progress",
+    "logoutButton": "Log out",
+    "navBarBusinessNJGovLink": "https://business.nj.gov/",
+    "needsAccountModalButtonText": "Link with myNewJersey",
+    "accountSuccessSnackbarBody": "You can now save your progress, access more features, and edit your business profile.",
+    "accountSuccessSnackbarTitle": "Welcome to your Business.NJ.gov account!",
+    "navBarUnnamedBusinessText": "Unnamed",
+    "navBarMyAccountText": "My Account",
+    "taskPageNavBarHeading": "Business Roadmap",
+    "needsAccountSnackbarTitleExistingAccount": "Link Your myNewJersey Account to Save Your Progress",
+    "needsAccountModalBody": "To use this feature, you need to complete your registration with the State of New Jersey's secure login system, MyNewJersey. This will ensure your privacy, and allow you to come back any time."
+  },
+  "navigationQuickLinks": {
```

Comment:
I broke this out from inside of the one above, because I think its easier to handle visually in the CMS